contracts/zeppelin/lifecycle/Killable.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.4.8;
+
+
+import "../ownership/Ownable.sol";
+
+
+/*
+ * Killable
+ * Base contract that can be killed by owner. All funds in contract will be sent to the owner.
+ */
+contract Killable is Ownable {
```

Comment:
@nberger we should refer all zeppelin changes on the zeppelin community, solve them there.